src/cascadia/ut_app/TerminalApp.UnitTests.vcxproj

```diff
@@ -52,6 +52,7 @@
     <ProjectReference Include="$(OpenConsoleDir)\src\cascadia\TerminalApp\TerminalAppLib.vcxproj" />
     <ProjectReference Include="$(OpenConsoleDir)\src\cascadia\TerminalSettingsModel\Microsoft.Terminal.Settings.ModelLib.vcxproj" />
     <ProjectReference Include="$(OpenConsoleDir)\src\types\lib\types.vcxproj" />
+    <ProjectReference Include="$(OpenConsoleDir)src\propslib\propslib.vcxproj" />

```

Comment:
Why do we need this?

Reply:
Same as the other one. The tests rely on TSM and now TSM needs information from unified propslib for registry operations on default app.